dozer/cogs/ftc.py

```diff
@@ -230,6 +271,19 @@ async def team(self, ctx: DozerContext, team_num: int):
                         value=', '.join((team_data['city'], team_data['stateProv'], team_data['country'])) or "Unknown")
             e.add_field(name='Org/Sponsors', value=team_data.get('nameFull', "").strip() or "_ _")
             e.add_field(name='Website', value=website or 'n/a')
```

Comment:
the `get_none_strip()` function is broken because `team_data['website']` can be `None` and that gets turned into `str(None)`. it's not strictly required but i would recommend fixing this by rewriting it to be:
```python
def get_none_strip(s, key):
    """Ensures that a get always returns a stripped string.""" 
    return str(s.get(key, "") or "").strip()
```